chatcontrol-bukkit/src/main/java/org/mineacademy/chatcontrol/model/Players.java

```diff
@@ -62,7 +62,7 @@ public static void showMotd(final WrappedSender wrapped, final boolean delay) {
 		final boolean firstTime = ((System.currentTimeMillis() - player.getFirstPlayed()) / 1000) < 5;
 		final int delayTicks = delay ? Settings.Motd.DELAY.getTimeTicks() : 3;
 
-		Platform.runTask(delayTicks, () -> {
+		Platform.runTaskAsync(delayTicks, () -> {
```

Comment:
I think I tried this quick hack in the past and there were side effects. I will add a check if the player is still online to prevent motd displaying for players who just disconnect